### PR TITLE
Project: Rails forms

### PR DESCRIPTION
**Pull request by rOluochKe**
- Implemented the rails forms following [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) tutorial. 
- We built the forms the normal way first and then the rails way

### CODE REVIEW

File: app/controllers/users_controller.rb [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class UsersController < ApplicationController
+  def new
+    @user = User.new
+  end
+
+  def create
+    # @user = User.new(name: params[:name], email: params[:email], password: params[:password])
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: You are to remove all commented out code. (Other than code specifically asked of you to be commented out for this specific project)

It’s better to delete commented code. If you need to review a previous version of your code, use git history

Please address this! You can see a [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/no_inline_comments.png) of this requirement from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/).

This should be no problem fixing 😄

File: app/controllers/users_controller.rb [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    @user = User.new(user_params)
+
+    if @user.save
+      redirect_to user_path(@user)
```
Comment (mcrd25) [inline app/controllers/users_controller.rb]: You ought to redirect to `new_user_path` as mentioned in the [Odin](https://www.theodinproject.com/courses/ruby-on-rails/lessons/forms) link which can be seen form this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png)

**Review by mcrd25 — CHANGES_REQUESTED** [review]
# Changes Required 🔍 👀 

Hi @rOluochKe and @ampaire 👋 ,

Y'all have done a great job with this project! 😄 
There are only 3 minor changes to address and you should get the 👍 

- Remove the commented out code mentioned in a review comment (Not the views form that were requested to be left commented out)
- Give your PR a descriptive title and description as can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/general/descriptive_prs.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/66243643/)
- There seems to be no show route but you are redirecting me there upon the successful creation of a user. See screenshot: 
Screen Shot 2020-01-07 at 8 55 01 AM
Maybe you should redirect to new user path upon successful creation and render new upon unsuccessful creation as suggested in the Odin project as can be seen from this [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/forms/user_create.png) as a requirement. Please Address 🙂 

You've done a great job! And once you've addressed the above issues there should be a clear path for you to move unto the ⏭ project!! 😄 

***Please makes changes and submit another review request.
If you have any questions, let me know here or on slack and I will be glad to clarify with you. You can also reach me and connect with me on my social media or email.***

*Happy coding 😄,*
**[--Maya](https://github.com/mcrd25)**
[GitHub](https://github.com/mcrd25) [Twitter](https://twitter.com/mcrd25) [linkedin](https://www.linkedin.com/in/mayadouglas/) [GitLab](https://gitlab.com/mcrd25)

---
*P.S. all requested changes not marked by [OPTIONAL] or are not positive comments are **REQUIRED***

**Comment by ampaire** [comment]
- Removed the commented out code
- Redirected the path for creating a new user to new user path
- Wrote a more descriptive PR for our project

**Review by uche-inyama — APPROVED** [review]
# Approved

Hi team everything is working fine now - you have implemented the changes requested by my colleague.

#### ``` Have a great day ``` :slightly_smiling_face: 

``` Uchechukwu Henry inyama ```

**Pull request merged**